### PR TITLE
Enable gazetteer button

### PR DESCRIPTION
**Pull request by Lenninlasd**
Hardcoded Gazetteer API: 
image

image

### Issue:
https://github.com/cga-harvard/worldmap/issues/13
https://github.com/cga-harvard/cga-worldmap/issues/225

### CODE REVIEW

**Comment by capooti** [comment]
Thanks. We have in production the other repository: could you send the PR there as well?

File: worldmap/templates/worldmap_client/worldmap.html [hunk 1]
```diff
@@ -226,15 +226,15 @@
                 	text: '<span class="x-btn-text">{% trans 'Share Map' %}</span>',
                 	iconCls: null,
                     actionTarget:  {target: "paneltbar", index: 24}
-                }{% if USE_GAZETTEER %},
```
Comment (capooti) [inline worldmap/templates/worldmap_client/worldmap.html]: why this? (though WM will always use anyway)

Comment (Lenninlasd) [inline worldmap/templates/worldmap_client/worldmap.html]: Fixed, It was a temporary workaround

**Comment by capooti** [comment]
sorry @Lenninlasd forget my comment. I thought this PR is vs cga-worldmap

**Pull request merged**